src/components/Navbar.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <div id="nav">
+      <div id="logo-warren">
+        <router-link to="/" id="logo-url">
+          <img src="/img/logoWarren.png" id="logo">
+        </router-link> 
+        <a>Warren</a>
+      </div>  
+      <router-link to="/">Sobre</router-link> 
+      <select name="submenu" id="menuInvestir">
+        <option hidden selected>Investir</option>
+        <option value="carteiras">Carteiras</option>
+        <option value="bolsa">Bolsa</option>
+        <option value="broker">Home Broker</option>
+      </select>
+      <!-- <router-link to="/">Investir</router-link> -->
+      <router-link to="/">Aprender</router-link>
+      <router-link to="/">Magazine</router-link>
```

Comment:
Não é muito indicado utilizar o router-link, mas sim utilizar um, this.$router.push() no botão